### DISCUSSION TITLE
Will students get access to get sponsored?

### DISCUSSION THREAD

**Discussion by TarunavBA** [github/india#123] [Students]
I am a student and am willing to get spónsored by others. And I am super excited to know if students get access to this?

**Comment by mvkaran** [comment]
Hi @TarunavBA. GitHub Sponsors is not yet fully available in India and has been opened up only to an initial group of developers. Once it has been fully rolled out, all GitHub users will be eligible to sign up for Sponsors.

**Reply by TarunavBA** [reply]
@mvkaran I have already applied for GitHub Sponsors.

**Reply by mvkaran** [reply]
@TarunavBA I understand that. Once GitHub Sponsors is available for India, you will be able to use it.

**Reply by TarunavBA** [reply]
@mvkaran GitHub sponsors is already available in India.
